Source/Core/DolphinQt/Settings/AudioPane.h

```diff
@@ -65,6 +66,9 @@ class AudioPane final : public QWidget
 #endif
 
   // Misc Settings
+  ConfigSlider* m_audio_buffer_size;
+  QLabel* m_audio_buffer_size_label;
```

Comment:
It looks like these don't need to be members. They can just be local variables inside `AudioPane::CreateWidgets`.